### PR TITLE
fix: misc fixes before shipping

### PR DESCRIPTION
**Pull request by craftablescience**
- set version
- set base dir properly on windows
- delete new'd memory at exit

### CODE REVIEW

**Review by ENDERZOMBI102 — APPROVED** [review]
LGTM!
is it ready to be merged?

**Comment by craftablescience** [comment]
Yep

File: src/main.cpp [hunk 1]
```diff
@@ -140,17 +141,16 @@ auto main( int argc, char* argv[] ) -> int {
 		excludes.emplace_back( "sdk_content.*" );
 		excludes.emplace_back( ".*.vmf_autosave" );
 		excludes.emplace_back( ".*index.rsv" );
```
Comment (SCell555) [inline src/main.cpp]: Should the dots in extensions be escaped?

Comment (ENDERZOMBI102) [reply src/main.cpp]: doesn't really matter, one could, but a single char can't do anything bad

**Pull request merged**